### PR TITLE
[App][Addinfo][1489120]

### PR DESCRIPTION
**Pull request by Rajeshwari-v**
Updated the document. Please review.

### CODE REVIEW

File: msteams-platform/build-your-first-app/build-and-run.md [hunk 1]
```diff
@@ -50,12 +49,14 @@ Once complete, there's a **Compiled successfully!** message in the terminal. You
 
 ## 4. Sideload your app in Teams
 
-Your app is ready to test in Teams. To do this, you must have an account that allows app sideloading. (If you aren't sure you have that, learn about getting a [Teams development account](../build-your-first-app/build-first-app-overview.md#set-up-your-development-account).)
+Your app is ready to test in Teams. To do this, you must have an account that allows app sideloading. If you are not sure you have that account, learn about getting a [Teams development account](../build-your-first-app/build-first-app-overview.md#set-up-your-development-account). Sideload your app in Teams using the following steps:
```
Comment (KirtiPereira) [inline msteams-platform/build-your-first-app/build-and-run.md:52]: "that account" is which account?

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
Please verify and update the comment.

**Review by Trinetra-MSFT — COMMENTED** [review]
Next steps does not look good
image

Why do we have that div

**Comment by Rajeshwari-v** [comment]
> Next steps does not look good
> image
> 
> Why do we have that div

@Trinetra-MSFT, Incorporated the suggested changes. Please review and approve.

**Review by Trinetra-MSFT — APPROVED** [review]

**Review by KirtiPereira — DISMISSED** [review]
Approve

File: msteams-platform/build-your-first-app/build-and-run.md [hunk 2]
```diff
@@ -50,12 +49,14 @@ Once complete, there's a **Compiled successfully!** message in the terminal. You
...
 
 > [!TIP]
-> Before sideloading your app, check for issues using the [validation feature in App Studio](../concepts/deploy-and-publish/appsource/prepare/submission-checklist.md#teams-app-validation-tool), which is included in the toolkit. Errors must be fixed to successfully sideload the app.
+> Check for issues before sideloading your app, using the [validation feature in App Studio](../concepts/deploy-and-publish/appsource/prepare/submission-checklist.md#teams-app-validation-tool), which is included in the toolkit. Fix the errors to successfully sideload the app.
 
-1. In Visual Studio Code, press the **F5** key to launch a Teams web client.
+> [!NOTE]
+> Follow the steps to [Turn on app sideloading](../concepts/build-and-test/prepare-your-o365-tenant.md#enable-custom-teams-apps-and-turn-on-custom-app-uploading), if the side loading is not enabled.
```
Comment (surbhigupta12) [inline msteams-platform/build-your-first-app/build-and-run.md:58]: Please follow style guide here.

**Review by surbhigupta12 — APPROVED** [review]
Approving the PR

**Review by KirtiPereira — APPROVED** [review]
Approve.

**Pull request merged**